### PR TITLE
Code lambdaization with Eclipse.

### PR DESCRIPTION
**Pull request by jolkdarr**
Hi again.
Since Java 8 is required, lambdas should be used to get  a code more compact.
Eclipse with Gradle support is a good mean to perform the task (many thanks to Eclipse Foundation).

**Note**.
Eclipse tries to process the class `DisplayNonmodelMessage` but fails because of a required forward declaration.
Test code not modified.

### CODE REVIEW

**Comment by jolkdarr** [comment]
Irrelevant theme file already removed.
Conflicting java files fixed (sorry for the inconvenience).
Note: `DisplayNonmodelMessage` class processed "manually".

File: spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java [hunk 1]
```diff
@@ -109,42 +109,26 @@ static void sleep(int seconds) {
     }
 
     /*
-    static Runnable moveToFrontLater = new Runnable() {
-        @Override
-        public void run() {
-            sleep(5);
-            SwingUtilities.invokeLater(moveToFront);
-        }
+    static Runnable moveToFront = () -> {
```
Comment (iloveeclipse) [inline spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java]: I don't see why it is needed to modify commented out code.

File: spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java [hunk 2]
```diff
@@ -109,42 +109,26 @@ static void sleep(int seconds) {
...
-        @Override
-        public void run() {
-            JFrame frame = messageFrame;
-            if (frame != null) {
-                frame.toFront();
-            }
-        }
+    static Runnable clearAlwaysOnTop = () -> {
+        JFrame frame = messageFrame;
+        if (frame != null)
```
Comment (iloveeclipse) [inline spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java]: please always use {} braces for ALL if/else/for/while etc constructs.

File: spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java [hunk 3]
```diff
@@ -445,21 +425,8 @@ private void examineCallSite(Location location, ConstantPoolGen cpg, TypeDataflo
             }
         }
          */
-        BitSet nullArgSet = frame.getArgumentSet(invokeInstruction, cpg, new DataflowValueChooser<IsNullValue>() {
-            @Override
-            public boolean choose(IsNullValue value) {
-                // Only choose non-exception values.
```
Comment (iloveeclipse) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java]: Please restore the comment.

Comment (jolkdarr) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java]: Modified by my PR ?
=> to restore.

File: spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java (hunk 2 above)
Comment (jolkdarr) [reply spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java]: I was not aware of that rule/recommendation => will be fulfilled from now on.

File: spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java (hunk 1 above)
Comment (jolkdarr) [reply spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java]: The modification is not required but is consistent with the modifications of similar methods.

File: spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java [hunk 4]
```diff
@@ -445,21 +425,11 @@ private void examineCallSite(Location location, ConstantPoolGen cpg, TypeDataflo
...
-                // Values null on an exception path might be due to
```
Comment (iloveeclipse) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java]: Sorry, I mean of course full comment, not just the first line. Also the line below :-)

Comment (jolkdarr) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java]: Eclipse relaunched... ;)

File: spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java [hunk 5]
```diff
@@ -428,6 +428,7 @@ private void examineCallSite(Location location, ConstantPoolGen cpg, TypeDataflo
 
         BitSet nullArgSet = frame.getArgumentSet(invokeInstruction, cpg,
                                                  // Only choose non-exception values.
+                                                 // Values null on an exception path might be due to
```
Comment (iloveeclipse) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindNullDeref.java]: Sorry, do I need to comment on every missing line? Next one please.

**Comment by jolkdarr** [comment]
It's 00:51 AM, that's not a time for contributing!

**Pull request merged**